### PR TITLE
Fix sonarcloud hotspots

### PR DESCRIPTION
**Pull request by schaubl**

### CODE REVIEW

**Review by 0xbase12 — CHANGES_REQUESTED** [review]
Changelog

File: code/src/nuvla/connector/nuvlabox.py [hunk 1]
```diff
@@ -586,7 +583,7 @@ def update_nuvlabox_engine(self, **kwargs):
         command = ['update']
 
         # action args
-        command.append('--quiet')
+        # command.append('--quiet')
```
[the remark this answers is not in the document]
Comment (schaubl) [reply code/src/nuvla/connector/nuvlabox.py:586]: yes

File: code/src/nuvla/connector/utils.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def utc_from_now_iso(seconds):
 
 
 def unique_id(*args):
-    return hashlib.md5(':'.join(map(str, args)).encode()).hexdigest()
+    return hashlib.sha256(':'.join(map(str, args)).encode()).hexdigest()
```
[the remark this answers is not in the document]
Comment (schaubl) [reply code/src/nuvla/connector/utils.py:39]: everything will be fine

**Comment by schaubl** [comment]
> Changelog

done

**Pull request merged**